rclpy/rclpy/parameter_service.py

```diff
@@ -65,7 +69,9 @@ def __init__(self, node):
             qos_profile=qos_profile_parameters
         )
 
-    def _describe_parameters_callback(self, request, response):
+    def _describe_parameters_callback(self, request: DescribeParameters.Request,
+                                      response: DescribeParameters.Response
+                                      ) -> DescribeParameters.Response:
```

Comment:
I don't particularly care for this style (even if it does pass our linters).  I think elsewhere we do:

```suggestion
    def _describe_parameters_callback(
        self,
        request: DescribeParameters.Request,
        response: DescribeParameters.Response
    ) -> DescribeParameters.Response:
```

(same goes for similar changes below)

Reply:
The linter seems to prefer

```python
def _describe_parameters_callback(
            self,
            request: DescribeParameters.Request,
            response: DescribeParameters.Response
            ) -> DescribeParameters.Response:
 ```
      
 over
      
 ```python
      def _describe_parameters_callback(
            self,
            request: DescribeParameters.Request,
            response: DescribeParameters.Response
          ) -> DescribeParameters.Response:
```

Reply:
See, for instance, https://github.com/ros2/rclpy/blob/adfcb2bd00a8301b310e4fd90731d61e94a86753/rclpy/rclpy/executors.py#L314-L318

Reply:
Ah I see.